### DISCUSSION TITLE
Laptop Power Management

### DISCUSSION THREAD

**Discussion by DanieleBertagnoli** [mylinuxforwork/dotfiles#175] [Q&A]
Hi, during the dotfiles installations I selected "install laptop power configurations" or something like that. Where I can find them?

**Comment by mylinuxforwork** [comment]
Hi. 
It is only a special configuration for touchpads.
This file should look slightly different on your system https://github.com/mylinuxforwork/dotfiles/blob/main/share/dotfiles/.config/hypr/conf/keyboard.conf

**Reply by DanieleBertagnoli** [reply]
Oh, so the laptop configuration was referred to the touchpad things. I thought that it was something to allow me to switch between power configurations (e.g. balance, performance and battery save mode). Is there something in the dotfiles which allows me to do so? Otherwise I'll look for them in the AUR repos. Thanks you!
